layouts/_default/baseof.html

```diff
@@ -164,13 +164,19 @@
 
         <main class="page-main">
             {{ block "main" . }}{{ end }}
-            {{- with .Site.Menus.secondary }}
-                {{- if gt (len .) 0 }}
-                    {{- partial "partials/menus/secondary.html" (dict "menu" .) -}}
-                {{- end }}
-            {{- end }}
         </main>
 
+        {{- with .Site.Menus.secondary }}
+            {{- if gt (len .) 0 }}
+                <nav class="section-container section-container--dimmed">
+                    <section class="section section--long">
```

Comment:
Do we need <section> tag here? If our secondary menu is divided into different categories or groups, we could use <section> to separate these categories, but we have a simple menu and together with <nav> it just makes things more complex ... I would use <div> here, or apply section styles to <nav>

Reply:
Good point. Changed it to `<div><nav>` where the `div` is just for styling purposes.